### ISSUE TITLE
Program hangs on startup (Trac #73)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#77]
This problem appears to have been introduced in commit b3973d8c7fe827bf1991a06f89a66f318ebb6242.

Tested on Windows 7.

Program debug log:

NOT FROZEN!
INFO: Root directory: D:\docs\ryan\dev\python\microdrop\microdrop
Loading chipmunk for Windows (32bit) [C:\Python27\lib\site-packages\pymunk\chipmunk.dll]
DEBUG: app_data_dir()=C:\Users\Ryan\AppData\Roaming
INFO: FrameGrabberChild is ready
INFO: request start: 2012-03-14 16:11:32.741000
DEBUG: [Config].load()
INFO: Loading config file from C:\Users\Ryan\AppData\Roaming\microdrop\microdrop.ini
DEBUG: get_skeleton_path(plugins)
DEBUG: common_app_data_dir()=C:\ProgramData
WARNING: warning: plugins does not exist in common AppData dir
DEBUG: emit_signal: microdrop.gui.main_window_controller.on_warning()
INFO: Loading plugins:
INFO: 	 d:\docs\Microdrop\plugins\dmf_control_board\microdrop
DEBUG: import plugins.dmf_control_board.microdrop
DEBUG: emit_signal: microdrop.gui.config_controller.on_app_init()
DEBUG: emit_signal: microdrop.gui.main_window_controller.on_app_init()
DEBUG: emit_signal: microdrop.gui.dmf_device_controller.on_app_init()

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/73
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "This problem appears to have been introduced in commit b3973d8c7fe827bf1991a06f89a66f318ebb6242.\n\nTested on Windows 7.\n\nProgram debug log:\n\nNOT FROZEN!\nINFO: Root directory: D:\\docs\\ryan\\dev\\python\\microdrop\\microdrop\nLoading chipmunk for Windows (32bit) [C:\\Python27\\lib\\site-packages\\pymunk\\chipmunk.dll]\nDEBUG: app_data_dir()=C:\\Users\\Ryan\\AppData\\Roaming\nINFO: FrameGrabberChild is ready\nINFO: request start: 2012-03-14 16:11:32.741000\nDEBUG: [Config].load()\nINFO: Loading config file from C:\\Users\\Ryan\\AppData\\Roaming\\microdrop\\microdrop.ini\nDEBUG: get_skeleton_path(plugins)\nDEBUG: common_app_data_dir()=C:\\ProgramData\nWARNING: warning: plugins does not exist in common AppData dir\nDEBUG: emit_signal: microdrop.gui.main_window_controller.on_warning()\nINFO: Loading plugins:\nINFO: \t d:\\docs\\Microdrop\\plugins\\dmf_control_board\\microdrop\nDEBUG: import plugins.dmf_control_board.microdrop\nDEBUG: emit_signal: microdrop.gui.config_controller.on_app_init()\nDEBUG: emit_signal: microdrop.gui.main_window_controller.on_app_init()\nDEBUG: emit_signal: microdrop.gui.dmf_device_controller.on_app_init()", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Program hangs on startup", 
    "priority": "blocker", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-03-14T20:18:27", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120315T03:47:30: 
* christian@fobel.net commented:

> commit 596b368585f6ae0cc3dcebcec98f6b47b184b09c
> Author: Christian Fobel <christian@fobel.net>
> Date:   Wed Mar 14 23:40:09 2012 -0400
> 
>     [SUBMODULE] Modify task_scheduler to return fir...
>     
>     [SUBMODULE] Modify task_scheduler to return first solution.  Fixes #77.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
